rudder-inventory-ldap/SPECS/rudder-inventory-ldap.spec

```diff
@@ -103,22 +104,40 @@ rudder-agent package installed) and for configuration rules and parameters.
 #=================================================
 %prep
 
+cp -rf %{_sourcedir}/berkeleydb-source %{_builddir}
 cp -rf %{_sourcedir}/openldap-source %{_builddir}
 
 #=================================================
 # Building
 #=================================================
 %build
-cd openldap-source
 
 # Ensure an appropriate environment for the compiler
 export CFLAGS="$RPM_OPT_FLAGS"
 export CXXFLAGS="$RPM_OPT_FLAGS"
 
-./configure --build=%_target --prefix=%{rudderdir} --enable-dynamic --enable-debug --enable-modules --enable-hdb=mod --enable-monitor=mod --enable-dynlist=mod --with-cyrus-sasl
+# 1 - BerkeleyDB
+cd berkeleydb-source/build_unix/
+../dist/configure --build=%_target --prefix=%{rudderdir}
+
+make %{?_smp_mflags}
+make install
+
+cd ../..
+
+# 2 - OpenLDAP
+cd openldap-source
+
+export LD_LIBRARY_PATH="/opt/rudder/lib"
+export CPPFLAGS="-I/opt/rudder/include"
+export LDFLAGS="-L/opt/rudder/lib"
+export LIBS="-L/opt/rudder/lib"
+
+./configure --build=%_target --prefix=%{rudderdir} --libdir=%{rudderdir}/lib/ldap --enable-dynamic --enable-debug --enable-modules --enable-hdb=mod --enable-monitor=mod --enable-dynlist=mod --with-cyrus-sasl
```

Comment:
To store all the OpenLDAP related libs in /opt/rudder/lib/ldap to make things cleaner :)